src/game/GameResults.as

```diff
@@ -1317,7 +1324,7 @@ package game
                 return;
 
             // Based on target
-            if (target == navSaveReplay && navSaveReplay.visible)
+            if (target == navSaveReplay && navSaveReplay.visible && !grayedOut)
```

Comment:
Should be able to just use the enabled status on the button instead of a separate variable.
```suggestion
            if (target == navSaveReplay && navSaveReplay.visible && navSaveReplay.enabled)
```